src/wrapper/theta.rs

```diff
@@ -133,8 +133,11 @@ pub struct ThetaIntersection {
 }
 
 impl ThetaIntersection {
-    /// Create a theta union over nothing, which corresponds to the
-    /// empty set.
+    /// Create a theta intersection over nothing, which corresponds to
+    /// an infinite set.
+    ///
+    /// Calls to `.sketch()` will return `None` unless `.merge()` has
```

Comment:
Did you observe this behavior? Presumably, merging two universal sets which have no keys should yield another universal set still.

What matters is that `update()` has been called (or we merged with something updated)

Reply:
The behavior I observed was that without calling `merge` at all (i.e. the intersection of “an empty set of sets”), `sketch` returned `None`. I didn't have a use case for this, I was just curious what would happen for the documentation.